python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -72,10 +72,27 @@ def _validateGalsimInterpolant(name: str) -> bool:
 
 
 class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
+    spatialOrderPerBand = pexConfig.DictField(
+        doc="Per-band spatial order for PSF kernel creation. "
+        "Ignored if piffPsfConfigYaml is set.",
+        keytype=str,
+        itemtype=int,
+        default={
+            "u": 2,
+            "g": 4,
+            "r": 4,
+            "i": 4,
+            "z": 4,
+            "y": 4,
```

Comment:
We should just set these in `obs_` packages.

Reply:
I thought we were supposed to set (expected) LSST values as the defaults and let the "other" `obs_` packages override if they wish?

Reply:
I guess me I am more on the side of just in case. I would be more in favor of having an overrides in obs_lsst anyway.